src/Mod/PartDesign/App/FeatureChamfer.cpp

```diff
@@ -115,7 +115,8 @@ App::DocumentObjectExecReturn *Chamfer::execute(void)
     }
 
     std::vector<std::string> SubNames = std::vector<std::string>(Base.getSubValues());
-    getContiniusEdges(TopShape, SubNames);
+    std::vector<std::string> FaceNames;
+    getContiniusEdges(TopShape, SubNames, FaceNames);

```

Comment:
Typo: `s/Continius/Continous`

Reply:
Thanks for the review. This typo was in the original code see #5031 where luzpaz has fixed this now.

I will rebase this PR with his fix soon.

Reply:
@plaes "continUous" replacing a typo by another isn't of great help generally.